### PR TITLE
feat: Eigenda disperser `/get/` and `/put/` endpoints

### PR DESCRIPTION
**Pull request by juan518munoz**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk_supervisor fmt` and `zk_supervisor lint`.

### CODE REVIEW

File: core/node/eigenda_proxy/src/errors.rs [hunk 1]
```diff
@@ -11,7 +13,17 @@ pub enum MemStoreError {
...
 }
+
+pub(crate) enum RequestProcessorError {
+    EigenDA(EigenDAError),
+}
+
+impl IntoResponse for RequestProcessorError {
+    fn into_response(self) -> Response {
+        unimplemented!("EigenDA request error into response")
+    }
```
Comment (gianbelinche) [inline core/node/eigenda_proxy/src/errors.rs:28]: Shouldn't this be implemented?

File: core/node/eigenda_proxy/src/lib.rs [hunk 2]
```diff
@@ -5,20 +5,37 @@ use std::net::SocketAddr;
...
+
     let bind_address = SocketAddr::from(([0, 0, 0, 0], 4242));
     tracing::debug!("Starting eigenda proxy on {bind_address}");
-    let app = create_eigenda_proxy_router();
+
+    let eigenda_client = match config {
+        EigenDAConfig::Disperser(disperser_config) => EigenDAClient::new(disperser_config)
+            .await
+            .map_err(|e| anyhow::anyhow!("Failed to create EigenDA client: {:?}", e))?,
+        _ => panic!("memstore unimplemented"),
```
Comment (gianbelinche) [inline core/node/eigenda_proxy/src/lib.rs:36]: Next would be implementing this for memstore also

File: core/node/eigenda_proxy/src/errors.rs (hunk 1 above)
Comment (juan518munoz) [reply core/node/eigenda_proxy/src/errors.rs]: You are correct, addressed in [680a207](https://github.com/lambdaclass/zksync-era/pull/313/commits/680a207627eef00ac5f619d6ee02e312b116c105)

File: core/node/eigenda_proxy/src/lib.rs (hunk 2 above)
Comment (juan518munoz) [reply core/node/eigenda_proxy/src/lib.rs:36]: Yes! Working on it

**Pull request closed without merging**